apps/point-of-sale/src/components/Cart/CartActionsComponent.vue

```diff
@@ -1,4 +1,7 @@
 <template>
+  <Message v-show="stopCheckoutForUserInDebt" severity="warn">
+    You are a user that cannot go into debt!
+  </Message>
```

Comment:
I think a pop-up would be better, since now you can also click it "away" which is weird. 

In my opinion a better solution would be that it shows a pop-up when trying to checkout instead?